jacorb-name-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM jboss/base:latest
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /opt/jboss
+EXPOSE 3528
+
+ENV JACORB_VERSION=2.3.1 \
+    JACORB_HOME=/opt/jboss/jacorb \
+    NAME_SERVER_PORT=3528 \
+    PATH=$PATH:/opt/jboss/jacorb/bin
+
+USER root
+
+RUN yum install -y ant \
+    && yum clean all
+
+USER jboss
+
+RUN curl -LOk http://www.jacorb.org/releases/$JACORB_VERSION/jacorb-$JACORB_VERSION-bin.zip \
```

Comment:
In this case there is no relation to WildFly 8.2.0.Final. This JTS server
goes standalone. How do you tag other projects which run standalone?

On Mon, Aug 24, 2015 at 10:40 AM, Marek Goldmann <notifications@github.com>
wrote:

> In jacorb-name-server/Dockerfile
> <https://github.com/jboss-dockerfiles/narayana/pull/2#discussion_r37734703>
> :
>
> > +WORKDIR /opt/jboss
> > +EXPOSE 3528
> > +
> > +ENV JACORB_VERSION=2.3.1 \
> > +    JACORB_HOME=/opt/jboss/jacorb \
> > +    NAME_SERVER_PORT=3528 \
> > +    PATH=$PATH:/opt/jboss/jacorb/bin
> > +
> > +USER root
> > +
> > +RUN yum install -y ant \
> > +    && yum clean all
> > +
> > +USER jboss
> > +
> > +RUN curl -LOk http://www.jacorb.org/releases/$JACORB_VERSION/jacorb-$JACORB_VERSION-bin.zip \
>
> It's not that easy, since we have tags globally for the repo and ship
> different images with different versions. If we start tagging things with
> the versions it'll become a mess, because how we differentiate between
> 8.2.0.Final tag we already have and the other tags you propose?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jboss-dockerfiles/narayana/pull/2/files#r37734703>.
>